src/adapter/src/coord/sequencer/inner/create_view.rs

```diff
@@ -122,7 +126,7 @@ impl Coordinator {
             plan,
             resolved_ids,
         }: CreateViewOptimize,
-        otel_ctx: OpenTelemetryContext,
+        parent_span: Span,
```

Comment:
Explicitly passing around the parent span is the only way I could figure out how to make the tree with the stages as siblings instead of children of each other. Is there a better way?

Reply:
Given `parent_span` here is the same span that we use in the `.instrument(...)` call on this method, I would expect that within this method `Span::current()` would return the same value as `parent_span`.

Because of that, and related to the above comment about cloning the span, I wouldn't think we need to pass it explicitly to this method. And instead we can use the span returned from `Span::current()` on line 143.

Then to make things siblings instead of children, see my comment below about calling `.exit()`.